src/main/java/org/coursesjava/services/ConfigService.java

```diff
@@ -0,0 +1,37 @@
+package org.coursesjava.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import org.coursesjava.model.ServerConfiguration;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class ConfigService {
+    private final ObjectMapper mapper = new XmlMapper();
+    private final File config = new File("src/main/resources/config.xml");
+    public void create() {
+        try {
+            mapper.writeValue(config, new ServerConfiguration());
+        } catch (IOException ex) {
+            System.err.println("File write error! " + ex.getMessage());
+            ex.printStackTrace();
```

Comment:
not a good practice to just print exception, in case if exception is happened, ConfigService will be in incorrect state, it will not work but object will be created